lore/learningobjects/models.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+
+
+class Course(models.Model):
+    repository = models.ForeignKey(Repository)
+    org = models.TextField()
+    course_number = models.IntegerField()
+    semester = models.TextField()
+    import_date = models.DateField()
+    imported_by = models.TextField()
+
+class LearningObject(models.Model):
+    course = models.ForeignKey(Course)
+    learning_object_type = models.ForeignKey(LearningObjectType)
+    uuid = models.TextField()
+    title = models.TextField()
+    description = models.TextField()
+    content_xml = models.TextField()
+    path_xml = models.TextField()
+    mpath = models.TextField()
+    url_path = models.TextField()
+    parent_id = models.ForeignKey("self")
```

Comment:
We may want to rename this 'parent' for consistency, not 100% sure on naming convention however